ui/pages/reservation-unit/[id].tsx

```diff
@@ -1049,140 +1043,105 @@ const ReservationUnit = ({
                     </p>
                   )}
                   {reservationUnit.reservationEnds && (
                     <p>
-                      <Trans i18nKey="reservationUnit:reservationInfo2">
-                        <strong>
-                          Varauskalenteri on auki{" "}
-                          {{
-                            reservationEnds: formatDate(
-                              reservationUnit.reservationEnds
-                            ),
-                          }}
-                        </strong>{" "}
-                        asti.
-                      </Trans>
+                      <Trans
+                        defaults="Varauskalenteri on auki <bold>{{reservationEnds}}</bold> asti."
+                        values={{
+                          reservationEnds: formatDate(
+                            reservationUnit.reservationEnds
+                          ),
+                        }}
+                        components={{ bold: <strong /> }}
+                      />
                     </p>
                   )}
                   {reservationUnit.minReservationDuration &&
                     reservationUnit.maxReservationDuration && (
                       <p>
-                        <Trans i18nKey="reservationUnit:reservationInfo3">
-                          Varauksen keston tulee olla välillä{" "}
-                          <strong>
-                            {{
-                              minReservationDuration: formatSecondDuration(
-                                reservationUnit.minReservationDuration,
-                                false
-                              ),
-                            }}
-                          </strong>{" "}
-                          ja{" "}
-                          <strong>
-                            {{
-                              maxReservationDuration: formatSecondDuration(
-                                reservationUnit.maxReservationDuration,
-                                false
-                              ),
-                            }}
-                          </strong>
-                          .
-                        </Trans>
+                        <Trans
+                          defaults="Varauksen keston tulee olla välillä <bold>{{minReservationDuration}}</bold> ja <bold>{{maxReservationDuration}}</bold>."
+                          values={{
+                            minReservationDuration: formatSecondDuration(
+                              reservationUnit.minReservationDuration,
+                              false
+                            ),
+                            maxReservationDuration: formatSecondDuration(
+                              reservationUnit.maxReservationDuration,
+                              false
+                            ),
+                          }}
+                          components={{ bold: <strong /> }}
+                        />
```

Comment:
Missing i18nKey. Need to refactor the translation tags.
See https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0